---
layout: page
title: 
---

### Submitted Manuscripts

**Michael I. Love**, John B. Hogenesch, Rafael A. Irizarry. Modeling of
RNA-seq fragment sequence bias reduces systematic errors in
transcript abundance estimation. (submitted)
[bioRxiv](http://biorxiv.org/content/early/2015/08/28/025767),
[alpine package](http://github.com/mikelove/alpine)

Leonardo Collado Torres, Alyssa C. Frazee, **Michael I. Love**, Rafael A.
Irizarry, Andrew E. Jaffe, Jeffrey Leek. derfinder: Software for
annotation-agnostic RNA-seq differential expression
analysis. (submitted)
[bioRxiv](http://biorxiv.org/content/early/2015/02/19/015370),
[derfinder package](http://bioconductor.org/packages/derfinder)

### Publications

Mingxiang Teng, **Michael I. Love**, Carrie A. Davis, Sarah Djebali,
Alexander Dobin, Brenton R. Graveley, Sheng Li, Christopher E. Mason,
Sara Olson, Dmitri Pervouchine, Cricket A. Sloan, Xintao Wei, Lijun
Zhan and Rafael A. Irizarry.
A benchmark for RNA-seq quantification pipelines. *Genome Biology*,
17(74), April 2016.
[doi: 10.1186/s13059-016-0940-1](http://dx.doi.org/10.1186/s13059-016-0940-1)

Charlotte Soneson, **Michael I. Love**, Mark D. Robinson. Differential
analyses for RNA-seq: transcript-level estimates improve gene-level
inferences. *F1000Research*,
4(1521), December 2015. 
[F1000Research](http://f1000research.com/articles/4-1521/v1),
[doi: 10.12688/f1000research.7563.1](http://dx.doi.org/10.12688/f1000research.7563.1),
[tximport package](https://github.com/mikelove/tximport)

**Michael I. Love**, Simon Anders, Vladislav Kim, Wolfgang Huber. RNA-seq
workflow: gene-level exploratory analysis and differential
expression. *F1000Research*, 4(1070), October 2015.
[doi: 10.12688/f1000research.7035.1](http://dx.doi.org/10.12688/f1000research.7035.1),
[live workflow](http://www.bioconductor.org/help/workflows/rnaseqGene/)

Raman Kumar, Mark A Corbett, Bregje WM van Bon, Alison Gardner, Joshua
A Woenig, Lachlan A Jolly, Evelyn Douglas, Kathryn Friend, Chuan Tan,
Hilde Van Esch, Maureen Holvoet, Martine Raynaud, Michael Field,
Melanie Leffler, Bartłomiej Budny, Marzena Wisniewska, Magdalena
Badura-Stronka, Anna Latos-Bieleńska, Jacqueline Batanian, Jill A
Rosenfeld, Lina Basel-Vanagaite, Corinna Jensen, Melanie Bienek, Guy
Froyen, Reinhard Ullmann, Hao Hu, **Michael I. Love**, Stefan A Haas, Pawel
Stankiewicz, Sau Wai Cheung, Anne Baxendale, Jillian Nicholl,
Elizabeth M Thompson, Eric Haan, Vera M Kalscheuer, Jozef Gecz.
Increased STAG2 dosage defines a novel cohesinopathy with intellectual
disability and behavioural problems. *Human Molecular Genetics*,
Epub aop, October 2015.
[doi: 10.1093/hmg/ddv414](http://dx.doi.org/10.1093/hmg/ddv414)

Stephan R. Starick, Jonas Ibn-Salem, Marcel Jurk, Céline Hernandez,
**Michael I. Love**, Ho-Ryun Chung, Martin Vingron, Morgane
Thomas-Chollier, and Sebastiaan H. Meijsing. ChIP-exo signal
associated with DNA-binding motifs provides insight into the genomic
binding of the glucocorticoid receptor and cooperating transcription
factors. *Genome Research*, 25(6):825–835, June 2015.
[doi: 10.1101/gr.185157.114](http://dx.doi.org/10.1101/gr.185157.114)

Wolfgang Huber, Vincent J. Carey, Robert Gentleman, Simon Anders, Marc
Carlson, Benilton S. Carvalho, Hector Corrada C. Bravo, Sean Davis,
Laurent Gatto, Thomas Girke, Raphael Gottardo, Florian Hahne, Kasper
D. Hansen, Rafael A. Irizarry, Michael Lawrence, **Michael I. Love**,
James MacDonald, Valerie Obenchain, Andrzej K.  Oleś, Hervé Pagès,
Alejandro Reyes, Paul Shannon, Gordon K. Smyth, Dan Tenenbaum, Levi
Waldron, and Martin Morgan. Orchestrating high-throughput genomic
analysis with Bioconductor. *Nature Methods*, 12(2):115–121,
February 2015.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4509590/),
[doi: 10.1038/nmeth.3252](http://dx.doi.org/10.1038/nmeth.3252)

H. Hu, S. A. Haas, J. Chelly, H. Van Esch, M. Raynaud, A. P. M.  de
Brouwer, S. Weinert, G. Froyen, S. G. M. Frints, F. Laumonnier,
T. Zemojtel, **Michael I. Love**, H. Richard, A. K. Emde, M. Bienek,
C. Jensen, M. Hambrock, U. Fischer, C. Langnick, M. Feldkamp,
W. Wissink-Lindhout, N. Lebrun, L. Castelnau, J. Rucci, R. Montjean,
O. Dorseuil, P. Billuart, T. Stuhlmann, M. Shaw, M. A. Corbett,
A. Gardner, S. Willis-Owen, C. Tan, K. L. Friend, S. Belet,
K. E. P. van Roozendaal, M. Jimenez-Pocquet, M. P. Moizard, N. Ronce,
R. Sun, S. O’Keeffe, R. Chenna, A. van Bommel, J. Goke, A. Hackett,
M. Field, L. Christie, J. Boyle, E. Haan, J. Nelson, G. Turner,
G. Baynam, G. Gillessen-Kaesbach, U. Muller, D. Steinberger, B. Budny,
M. Badura-Stronka, A. Latos-Bielenska, L. B. Ousager, P. Wieacker,
G. Rodriguez Criado, M. L. Bondeson, G. Anneren, A. Dufke, M. Cohen,
L. Van Maldergem, C. Vincent-Delorme, B. Echenne, B. Simon-Bouy,
T. Kleefstra, M. Willemsen, J. P. Fryns, K. Devriendt, R. Ullmann,
M. Vingron, K. Wrogemann, T. F. Wienker, A. Tzschach, H. van Bokhoven,
J. Gecz, T. J. Jentsch, W. Chen, H. H. Ropers, and
V. M. Kalscheuer. X-exome sequencing of 405 unresolved families
identifies seven novel intellectual disability genes. *Molecular
Psychiatry*, Epub aop, February 2015.
[doi: 10.1038/mp.2014.193](http://dx.doi.org/10.1038/mp.2014.193)

**Michael I. Love**, Wolfgang Huber, and Simon Anders. Moderated
estimation of fold change and dispersion for RNA-seq data with
DESeq2. *Genome Biology*, 15(12):550+, December 2014.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049/),
[doi: 10.1186/s13059-014-0550-8](http://dx.doi.org/doi:10.1186/s13059-014-0550-8),
[DESeq2 package](http://bioconductor.org/packages/DESeq2)

Wei Li, Han Xu, Tengfei Xiao, Le Cong, **Michael I. Love**, Feng
Zhang, Rafael A. Irizarry, Jun S. Liu, Myles Brown, and
X. S. Liu. MAGeCK enables robust identification of essential genes
from genome-scale CRISPR/cas9 knockout screens. *Genome Biology*,
15(12):554+, December 2014.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290824/),
[doi: 10.1186/s13059-014-0554-4](http://dx.doi.org/10.1186/s13059-014-0554-4)

Jonas Ibn-Salem, Sebastian Köhler, **Michael I. Love**, Ho-Ryun Chung,
Ni Huang, Matthew E. Hurles, Melissa Haendel, Nicole L. Washington,
Damian Smedley, Christopher J. Mungall, Suzanna E. Lewis, Claus-Eric
Ott, Sebastian Bauer, Paul N. Schofield, Stefan Mundlos, Malte
Spielmann, and Peter N. Robinson. Deletions of chromosomal regulatory
boundaries are associated with congenital disease. *Genome Biology*,
15(9):423+, September 2014.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4180961/),
[doi: 10.1186/s13059-014-0423-1](http://dx.doi.org/10.1186/s13059-014-0423-1)

Owen D. Solberg, Edwin J. Ostrin, **Michael I. Love**, Jeffrey
C. Peng, Nirav R. Bhakta, Lydia Hou, Christine Nguyen, Margaret Solon,
Cindy Nguyen, Andrea J. Barczak, Lorna T. Zlock, Denitza P. Blagev,
Walter E. Finkbeiner, K. Mark Ansel, Joseph R. Arron, David J. Erle,
and Prescott G. Woodruff. Airway epithelial miRNA expression is
altered in asthma. *American Journal of Respiratory and Critical Care
Medicine*, 186(10):965–974, November 2012.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530212/),
[doi: 10.1164/rccm.201201-0027OC](http://dx.doi.org/10.1164/rccm.201201-0027OC)

Ruping Sun, **Michael I. Love**, Tomasz Zemojtel, Anne-Katrin Emde,
Ho-Ryun Chung, Martin Vingron, and Stefan A. Haas. Breakpointer: using
local mapping artifacts to support sequence breakpoint discovery from
single-end reads. *Bioinformatics*, 28(7):1024–1025, April 2012.
[doi: 10.1093/bioinformatics/bts064](http://dx.doi.org/10.1093/bioinformatics/bts064)

**Michael I. Love**, Alena Myšičková, Ruping Sun, Vera Kalscheuer,
Martin Vingron, and Stefan A. Haas. Modeling read counts for CNV
detection in exome sequencing data. *Statistical Applications in
Genetics and Molecular Biology*, 10(1), January 2011.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517018/),
[doi: 10.2202/1544-6115.1732](http://dx.doi.org/10.2202/1544-6115.1732),
[exomeCopy package](http://bioconductor.org/packages/exomeCopy)

